lib/irb/debug.rb

```diff
@@ -81,6 +81,7 @@ def configure_irb_for_debugger(irb)
         IRB.instance_variable_set(:@debugger_irb, irb)
         irb.context.with_debugger = true
         irb.context.irb_name += ":rdbg"
+        irb.context.io.load_history
```

Comment:
It looks like we need some safeguarding here as `context.io` doesn't always include `HistorySavingAbility`, especially in some non-tty environments.

```suggestion
        irb.context.io.load_history if irb.context.io.class < HistorySavingAbility
```